### DISCUSSION TITLE
DateTime vs DateTimeOffset in time series

### DISCUSSION THREAD

**Discussion by brunomlopes** [ravendb/ravendb#13054] [Q&A] [answered]
I noticed that documentation and blogposts for timeseries use DateTime, like the one at https://ayende.com/blog/195297-C/ravendb-5-3-features-incremental-time-series-throttling-and-rate-limits?Key=102b4eb8-ea20-40bb-bd3d-74a2f19a4198 and docs at https://ravendb.net/docs/article-page/5.2/csharp/document-extensions/timeseries/client-api/operations/get .

Is there a specific reason that can be shared for using DateTime instead of using DateTimeOffset ?
I tend to always use DateTimeOffset to be more explicit about offsets, and am wondering if I'm missing something.

Thank you

**Comment by ayende** [comment, marked as answer]
The time series dates are always using `UTC` values, the problem is that even `DateTimeOffset` doesn't provide good information about the timepoint involved.
For example, when seeing `01:59:59.0000+02:00`  - what is the next second? Well, in different locations and different times, it may give different results. You have the _offset_, not the timezone. that isn't sufficient.

For that reason, we require that the data will be in `UTC` date time.
